rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -898,6 +906,8 @@ def process(self, message: Message, **kwargs: Any) -> None:
 
             message.set(ENTITIES, entities, add_to_output=True)
 
+        return out.get(DIAGNOSTIC_DATA)
```

Comment:
The intended use is to load functions from the rasa package, so explicitly importing tensorflow is not necessary. E.g. in [Vincent's script](https://github.com/RasaHQ/STAR-bots/blob/efe062a5da3d85c7dd7eef1540c7dd70a5ab16bc/assistants/directions/atw-explore.py#L129) he just calls `diagnostics["attention_weights"].numpy()` and never imports tensorflow directly. 

Calling `.numpy()` on the rasa side gives `AttributeError: 'Tensor' object has no attribute 'numpy'` because we're not in eager execution mode by default. I don't understand, though, why calling it in the script then works, however. @koaning do you have an idea?

Reply:
I'm assuming that TensorFlow assumes eager mode unless specified otherwise. My guess (guess!) is that Rasa prefers to turn it off because of the speedup that you can get. The last time I've done any real TensorFlow was back when 1.8 was "new" though. @Ghostvv might know for sure. 

That said, I can live with calling numpy on my script side.

Reply:
Ah! What was puzzling me is that eager execution is turned off in `rasa shell` but not when you call stuff from your script. But you're not calling `rasa shell` and rasa must turn off eager mode before anything that you'd use gets created. Makes sense.

Reply:
where do you call `.numpy`? it should happen in predict method not in tf code. so when you get smth from `out`, you should call `.numpy` on them, so that in Vincent code, he could use them as is

Reply:
Got it. Returning numpy arrays now.